### PR TITLE
debug habitat urdf + improve physics

### PR DESCRIPTION
**Pull request by cpaxton**

### CODE REVIEW

**Comment by cpaxton** [comment]
Screenshot from 2023-02-21 14-47-20
Screenshot from 2023-02-21 14-47-27
Screenshot from 2023-02-21 14-47-30
Modified wireframe + geometry - agent now sits on top of terrain properly

**Comment by cpaxton** [comment]
Screenshot from 2023-02-21 15-13-30
final update to add collision geometry for wheels as cylinders

File: test_pybullet.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
 
 # Load the stretch
 #robot_id = pb.loadURDF('./urdf/stretch.urdf', start_pos, start_orn)
-robot_id = pb.loadURDF('./urdf/hab_stretch_dex_wrist_simplified.urdf', start_pos, start_orn)
+#robot_id = pb.loadURDF('./urdf/hab_stretch_dex_wrist_simplified.urdf', start_pos, start_orn)
```
Comment (exhaustin) [inline test_pybullet.py:17]: Should we remove these comments?

**Review by exhaustin — APPROVED** [review]
Looks great, thanks for the detailed description of the changes along with visuals!

File: test_pybullet.py (hunk 1 above)
Comment (cpaxton) [reply test_pybullet.py:17]: this is a debugging script and this reminds me of the different locations for important files, so i'd honestly rather not

**Pull request merged**